components/ongage/ongage.app.js

```diff
@@ -0,0 +1,67 @@
+const axios = require('axios');
+const Ongage = require('ongage');
+
+module.exports = {
+  type: 'app',
+  app: 'ongage',
+  methods: {
+    _ongage (client) {
+      return new Ongage[client](
+        this.$auth.x_username,
+        this.$auth.x_password,
+        this.$auth.x_account_code
+      );
+    },
+    async _execute ({ body, ...config }) {
+      if (body) config.data = body;
+      const res = await axios(config);
+      return res.data;
+    },
+    async getLists (page) {
+      const api = this._ongage('ListsApi');
+      const req = api.getAll({ sort: 'name', order: 'ASC', limit: 50, offset: 50 * page });
+      return await this._execute(req);
+    },
+    async subscribe (listId, email, fields = {}, overwrite = false) {
+      const api = this._ongage('ContactsApi');
+      const req = api.create({ email, overwrite, fields }, listId);
+      return await this._execute(req);
+    },
+    async updateSubscriber (listId, email, fields = {}) {
+      const api = this._ongage('ContactsApi');
+      const req = api.update({ email, fields }, listId);
+      return await this._execute(req);
+    },
+    async findSubscriber (email) {
+      const api = this._ongage('ContactsApi');
+      const req = api.getListsByEmail(email);
+      return await this._execute(req);
+    }
+  },
+  propDefinitions: {
+    listId: {
+      type: 'string',
+      label: 'List ID',
+      async options ({ page }) {
+        const { payload } = await this.getLists(page);
+        return payload.map(list => ({ label: list.name, value: list.id }));
+      }
+    },
+    overwrite: {
```

Comment:
Typically we'll also add a `label` and `description` to each prop. Would you be able to add those for these? `description` supports Markdown, if you want to add formatting. See all of the options here: https://pipedream.com/docs/components/api/#user-input-props